samples/ChatSample/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "chatsample",
     "private": true,
     "dependencies": {
-        "@aspnet/signalr": "^1.0.0-rc1-final",
+        "@aspnet/signalr": "^1.0.0",
         "@aspnet/signalr-protocol-msgpack": "^1.0.0-rc1-final"
```

Comment:
Update this

Reply:
Why is thing happening... 😕 
I will fix it.